e2e/rbd.go

```diff
@@ -1330,6 +1348,12 @@ var _ = Describe("RBD", func() {
 			// NOTE: RWX is restricted for FileSystem VolumeMode at ceph-csi,
 			// see pull#261 for more details.
 			By("Create RWX+Block Mode PVC and bind to multiple pods via deployment using rbd-nbd mounter", func() {
+				if !testNBD {
+					e2elog.Logf("skipping NBD test")
+
+					return
```

Comment:
return will skip all the tests or only this test?

Reply:
It returns from the current function, so it should continue with the next test. The `Skip()` functions cause a more complete return, so those can not be used here.